scripts/src/middleware.ts

```diff
@@ -129,6 +138,12 @@ function serverErrorHandler(error: any, req: express.Request, res: express.Respo
 		message += `message: ${ error.toString() }`;
 	}
 
+	if (serverErrorTimeStamps.length === RESTART_ERROR_COUNT) {
```

Comment:
if you initialize `serverErrorTimeStamps` to 20 zeros, you can remove this `if`

Reply:
We can minify the code if we just want to make it less readable.